src/Sil/Bundle/ContactBundle/Resources/views/list/contact-addresses.html.twig

```diff
@@ -0,0 +1,24 @@
+<ul id="address-list" class="ui divided list">
+    {% if contact.defaultAddress %}
+        {{ include('SilContactBundle:block:address-block.html.twig', {
+            'isDefaultAddress': true,
+            'address': contact.defaultAddress
+        }) }}
+
+        {% for address in contact.addresses %}
+            {% if address.id != contact.defaultAddress.id %}
```

Comment:
you can compare `address` with `contact.defaultAddress` without checking their IDs.
e.g. : `{% if address is same as contact.defaultAddress %}`